trailblazer/store/crud/read.py

```diff
@@ -36,6 +38,27 @@ def get_nr_jobs_with_status_per_category(
         categories = categories.group_by(Job.name).all()
         return [{"name": category.name, "count": category.count} for category in categories]
 
+    def get_analyses_by_status_started_at_and_comment(
+        self, before: datetime = None, comment: str = None, status: str = None, limit: int = None
+    ) -> Optional[List[Analysis]]:
+        """Return analysis meeting supplied arguments."""
```

Comment:
```suggestion
        """Return analyses meeting supplied arguments. Returns only up to a limit if specified"""
```
I think this function does a bit more that can be gauged from the function name and docstring. Maybe worthwhile to split it up? I guess it will be a tradeoff.

Reply:
I agree that we could leave the limit part out of this function

Reply:
Now limit on the caller side